### PR TITLE
git-bulk: add examples

### PR DESCRIPTION
**Pull request by KMohZaid**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 6.4.0

### CODE REVIEW

**Comment by KMohZaid** [comment]
Mentioned in #5137

**Comment by KMohZaid** [comment]
**found duplicate PR #8946**

**Comment by navarroaxel** [comment]
@KMohZaid feel free to update this PR if you want to improve https://github.com/tldr-pages/tldr/blob/main/pages/common/git-bulk.md

**Comment by KMohZaid** [comment]
> @KMohZaid feel free to update this PR if you want to improve https://github.com/tldr-pages/tldr/blob/main/pages/common/git-bulk.md

Thanks

File: pages/common/git-bulk.md [hunk 1]
```diff
@@ -23,3 +27,11 @@
 - Run a Git command on the repositories of the current workspace:
 
 `git bulk {{command}} {{command_arguments}}`
+
+- Remove a registered workspace:
```
Comment (navarroaxel) [inline pages/common/git-bulk.md:31]:
```suggestion
- Remove a specific workspace:
```

File: pages/common/git-bulk.md [hunk 2]
```diff
@@ -23,3 +27,11 @@
...
+
+`git bulk --removeworkspace {{workspace_name}}`
+
+- Remove all registered workspaces:
```
Comment (navarroaxel) [inline pages/common/git-bulk.md:35]:
```suggestion
- Remove all workspaces:
```

**Review by navarroaxel — APPROVED** [review]

**Review by HoarderKing — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

> Welcome to tldr btw

**Pull request merged**